packages/connect/e2e/karma.config.js

```diff
@@ -125,7 +125,7 @@ module.exports = config => {
                 // replace ws module used in ./tests/websocket-client.js
                 new webpack.NormalModuleReplacementPlugin(
                     /ws$/,
-                    '@trezor/blockchain-link/lib/utils/ws',
+                    '@trezor/blockchain-link/src/utils/ws',
```

Comment:
just note for me, lets make sure that connect (web) tests got triggered

Reply:
ha, as of now, they did not 🤔 
<img width="855" alt="image" src="https://github.com/trezor/trezor-suite/assets/30367552/b83aea54-2a98-451e-9ab6-0ad39eda662a">

Reply:
how to run it?

Reply:
rules are as follow, at the moment. https://github.com/trezor/trezor-suite/blob/connec/interaction-less-fw-update/.github/workflows/connect-test.yml#L5-L17 

I guess there is something wrong about it. I am still not very knowledgable, maybe @vdovhanych knows

Reply:
the connect test were triggered i see them but they are cancelled in there